### PR TITLE
Media file and type validation

### PR DESCRIPTION
**Pull request by xNordo**
| Q               | A
| --------------- | -----
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| License         | MIT

### CODE REVIEW

File: src/Resources/config/validation/Media.yml [hunk 1]
```diff
@@ -4,6 +4,11 @@ BitBag\SyliusCmsPlugin\Entity\Media:
             fields: [code]
             groups: ['bitbag']
             message: 'bitbag_sylius_cms_plugin.media.code.unique'
+        - BitBag\SyliusCmsPlugin\Validator\FileMatchesType:
```
Comment (leszczuu) [inline src/Resources/config/validation/Media.yml:7]: This needs to be 'Constraint' not Validator

File: src/Validator/FileMatchesType.php [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace BitBag\SyliusCmsPlugin\Validator;
+
+use Symfony\Component\Validator\Constraint;
+
+
```
Comment (damonsson) [inline src/Validator/FileMatchesType.php:9]: Why 2 empty lines?

File: src/Validator/FileMatchesType.php [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+final class FileMatchesType extends Constraint
+{
+    public $messageImage = 'This file cannot be uploaded as an image';
+
+    public $messageVideo = 'This file cannot be uploaded as an video';
+
+    public $field;
```
Comment (damonsson) [inline src/Validator/FileMatchesType.php:16]: Where are variable types?

File: src/Validator/FileMatchesType.php [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+    public function getTargets()
```
Comment (damonsson) [inline src/Validator/FileMatchesType.php:18]: Return type?

**Review by leszczuu — APPROVED** [review]

**Pull request merged**